src/apps/bm_devkit/sdi12_example/user_code/user_code.cpp

```diff
@@ -154,7 +155,7 @@ void loop(void) {
     transmit_samples();
     current_sample_index = 0;
   }
-  vTaskDelay(pdMS_TO_TICKS(2000));
+//  vTaskDelay(pdMS_TO_TICKS(2000));
```

Comment:
Also this? I feel like if its not needed, maybe it is best to remove to avoid clutter and confusion.

Reply:
I shall delete it